api/app/build.gradle.kts

```diff
@@ -89,6 +89,7 @@ detekt {
     config =
         files("$projectDir/config/detekt.yml") // point to your custom config defining rules to run, overwriting default behavior
     baseline = file("$projectDir/config/baseline.xml") // a way of suppressing issues before introducing detekt
+    autoCorrect = true
```

Comment:
this should be alright to leave in CI as well. The task does fail even if it auto corrects and if we run this, it'll fix linting issues wherever it can